examples/rstudio/compare.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+percent_diff=$(compare -metric MSE "$1" "$2" null: 2>&1 > /dev/null | awk '{print $1}')
+if [[ $percent_diff == 0 ]];then
```

Comment:
```suggestion
if [[ $percent_diff = '0' ]];then
```

to emphasize that it's a string compare, not numeric compare. Also the shell comparison operator is a single equals (`==` is a Bash extension and will work but the rest of the code uses `=`).